crates/ide/src/hover.rs

```diff
@@ -136,11 +136,12 @@ pub(crate) fn hover(
         .flatten()
         .unique_by(|&(def, _)| def)
         .filter_map(|(def, node)| hover_for_definition(sema, file_id, def, &node, config))
-        .reduce(|mut acc, HoverResult { markup, actions }| {
+        .reduce(|mut acc: HoverResult, HoverResult { markup, actions }| {
```

Comment:
I added this type annotation here because I noticed r-a was not able to infer the type of `acc` without it :(